app/controllers/admin/bookings_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Admin::BookingsController < Admin::AdminController
+  include Admin::BookingsHelper
+
+  before_action :set_booking, only: %i(show edit update)
+  def index
+    @pagy, @bookings = pagy(Booking.all,
```

Comment:
Booking nên được sắp xếp theo thứ tự mới nhất và theo status
ví dụ status đang chờ xét duyệt sẽ phải hiển thị lên trước status đã thanh toán
Như vậy sẽ giúp người dùng duyệt booking nhanh hơn